org.eclipse.scanning.points/src/org/eclipse/scanning/points/CompoundIterator.java

```diff
@@ -24,91 +25,76 @@
  * compound generators by doing this. Otherwise the createPoints(...)
  * would do.
  *
+ * TODO Matt D. 2017-10-26 should we do all points generation in the jython layer? If so,
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)